examples/head_or_tails.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="h!")
+
+@bot.command()
+async def headortails(ctx, answer=None):
+    number = random.randint(0,1)
+
+    tails = 0
+    heads = 1
+
+    if answer is None:
+        await ctx.send("You have not set an argumeny, please do: h!headsortails `heads` or h!headsortails `tails`")
+    else:
+        if answer == "heads" or answer == "tails":
+            if number == tails:
```

Comment:
It would be better to hardcode the values here

Reply:
what you want! :)

Reply:
I added the things you wanted. I hope it's good now!